source/exploring_catalog_and_datasets/01_looking_up_the_catalog/catalog.rst

```diff
@@ -1,79 +1,179 @@
+.. image:: images/catalog_fullinterface.png
+   :alt: Full interface of the catalog, with both sides
 
-.. image:: images/explore_card.png
-   :alt: Dataset card
+On the left side (1), are displayed:
 
-On this example, the card is split in two parts. On the left part, there is a theme pictogram, the title, a description, the language and license, keywords and the original publisher. On the right part, there are links to access the data table, a map and analysis tools, and to export or access the API of the dataset.
+- the number of datasets currently displayed,
+- a catalog sort option to change the order of the datasets,
+- a variety of catalog filters to find specific datasets,
+- different possibilities to download the catalog.
 
-Like most of the catalog's elements, this card is fully customizable.
+On the right side (2), are displayed the datasets. Each dataset is represented by a rectangle called a "catalog card". On each card, the dataset's main metadata are displayed, along with quick links to visualize the data.
+
+Sorting the catalog
+-------------------
+
+By default, a dataset catalog is sorted by last modification date (i.e. the most recently modified datasets are displayed in first position, at the top of the catalog).
+
+It is possible to change in which order the catalog cards are displayed with the catalog sorting option below the number of datasets, on the left side of the interface.
+
+.. image:: images/catalog_sort.png
+   :alt: Catalog sorting option
+
+The catalog can be sorted by 10 different methods:
+
+- *Alphabetically* (one of the 3 most popular methods "A-Z"): the datasets are sorted by their titles, in an alphabetical order.
+- *Unalphabetically*: the datasets are sorted by their titles, in a reverse alphabetical order.
+- *Recently modified first* (one of the 3 most popular methods "Modified"): the datasets are sorted from the most recently modified to the least recently modified.
+- *Least mostly modified first*: the datasets are sorted from the least recently modified to the most recently modified.
+- *Most records first*: the datasets are sorted from the one with the biggest amount of records to the one with the least records.
+- *Least records first*: the datasets are sorted from the one with the least records to the one with the biggest amount of records.
+- *Most downloads first*: the datasets are sorted from the most downloaded one to the least downloaded.
+- *Least downloads first*: the datasets are sorted from the least downloaded one to the most downloaded.
+- *Most popular first* (one of the 3 most popular methods "Popular"): the datasets are sorted from the most popular one to the least popular.
+- *Least popular first*: the datasets are sorted from the least popular one to the least popular.
+
+To modify the sorting method, 2 possibilities:
+
+A. Click one of the 3 most popular methods: "Modified", "Popular", "A-Z".
+B. 1. Click the |icon-seemore| button to display all available sorting methods.
```

Comment:
For an unknown reason, the second level of the list shows as A. B. as well when the documentation is compiled. Should stay 1. 2. as this is indeed a sequence of actions.

Reply:
Thank you for pointing out this error, I managed to fix it!